<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderDetailDialog</class>
 <widget class="QDialog" name="OrderDetailDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>610</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>608</width>
    <height>610</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>608</width>
    <height>610</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Thông tin đơn hàng</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../resources.qrc">
    <normaloff>:/images/icon/note.png</normaloff>:/images/icon/note.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#OrderDetailDialog {
	background-color: #f5f5fa;
}</string>
  </property>
  <widget class="QLabel" name="label_41">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>12</y>
     <width>608</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 24px;
font-weight: bold;
color: #f27d85;</string>
   </property>
   <property name="text">
    <string>Chi tiết đơn hàng</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="wrapper" native="true">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>53</y>
     <width>571</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
	font-size: 16px;
	background-color: white;
	border-radius: 20px;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QWidget" name="widget_32" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_25" stretch="0,1,0,0">
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="label_54">
         <property name="styleSheet">
          <string notr="true">font-weight: bold;
color: #777;</string>
         </property>
         <property name="text">
          <string>Mã đơn hàng:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="orderID">
         <property name="text">
          <string>O1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_69">
         <property name="styleSheet">
          <string notr="true">font-weight: bold;
color: #777;</string>
         </property>
         <property name="text">
          <string>Ngày:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="date">
         <property name="text">
          <string>15/11/2021</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_40" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_35" stretch="0,1">
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="label_71">
         <property name="styleSheet">
          <string notr="true">font-weight: bold;
color: #777;</string>
         </property>
         <property name="text">
          <string>Người bán:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="sellerName">
         <property name="text">
          <string>Someone</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_41" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_27" stretch="0,1,0,0">
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="label_57">
         <property name="styleSheet">
          <string notr="true">font-weight: bold;
color: #777;</string>
         </property>
         <property name="text">
          <string>Người mua:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="buyerName">
         <property name="text">
          <string>Hoàng Trọng Vũ</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_74">
         <property name="styleSheet">
          <string notr="true">font-weight: bold;
color: #777;</string>
         </property>
         <property name="text">
          <string>Số điện thoại:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="buyerPhoneNumber">
         <property name="text">
          <string>0123456789</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_42" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_36" stretch="0,1">
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="label_72">
         <property name="styleSheet">
          <string notr="true">font-weight: bold;
color: #777;</string>
         </property>
         <property name="text">
          <string>Trạng thái:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="status">
         <property name="text">
          <string>Đã duyệt</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_49" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_43" stretch="0,1">
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="label_73">
         <property name="styleSheet">
          <string notr="true">font-weight: bold;
color: #777;</string>
         </property>
         <property name="text">
          <string>Đánh giá:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="rate">
         <property name="text">
          <string>Chưa đánh giá</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_43" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_37" stretch="0,1,0,0">
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="label_76">
         <property name="styleSheet">
          <string notr="true">font-weight: bold;
color: #777;</string>
         </property>
         <property name="text">
          <string>Sản phẩm:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="productName">
         <property name="text">
          <string>You are very beautiful</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_77">
         <property name="styleSheet">
          <string notr="true">font-weight: bold;
color: #777;</string>
         </property>
         <property name="text">
          <string>Số lượng:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="productQuantity">
         <property name="text">
          <string>1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_44" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_38" stretch="0,1">
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="label_78">
         <property name="styleSheet">
          <string notr="true">font-weight: bold;
color: #777;</string>
         </property>
         <property name="text">
          <string>Địa chỉ giao hàng:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="address">
         <property name="text">
          <string>Quảng Trị, Việt Nam</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignJustify|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_45" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_39" stretch="0,1">
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="label_79">
         <property name="styleSheet">
          <string notr="true">font-weight: bold;
color: #777;</string>
         </property>
         <property name="text">
          <string>Ghi chú giao hàng:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="note">
         <property name="text">
          <string>Không có ghi chú</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_46" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_40" stretch="0,1">
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="label_80">
         <property name="styleSheet">
          <string notr="true">font-weight: bold;
color: #777;</string>
         </property>
         <property name="text">
          <string>Hình thức thanh toán:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="payOption">
         <property name="text">
          <string>Thanh toán khi nhận hàng</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_48" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_42" stretch="0,1">
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="label_82">
         <property name="styleSheet">
          <string notr="true">font-weight: bold;
color: #777;</string>
         </property>
         <property name="text">
          <string>Voucher:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="voucher">
         <property name="text">
          <string>%</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_47" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_41" stretch="0,1">
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="label_81">
         <property name="styleSheet">
          <string notr="true">font-weight: bold;
color: #777;</string>
         </property>
         <property name="text">
          <string>Tổng tiền:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="totalCost">
         <property name="styleSheet">
          <string notr="true">font-weight: bold;
font-size: 18px;
color: #f27d85;</string>
         </property>
         <property name="text">
          <string>100.000đ</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="orderDetailClose">
   <property name="geometry">
    <rect>
     <x>248</x>
     <y>566</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #f2758d;
font-size: 16px;
color: white;
font-weight: bold;
padding: 6px 10px;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>Đóng</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
